test/nitro/claim-ts.test.ts

```diff
@@ -234,7 +187,7 @@ describe("claim (typescript)", function () {
     );
   });
 
-  it("Can claim with an empty exit request", async function () {
+  it("guarantor_funding == target_funding < outcome_sum, pay out all via empty exit request", async function () {
```

Comment:
what's the difference between "no exit requests" and "empty exit request"?

Reply:
On closer inspection, the two test cases appear to be the same. One of the test cases has been removed in fba7b2a.